src/alpaca_eval/models_configs/phi-2/prompt.txt

```diff
@@ -0,0 +1,2 @@
+A instruction given to an artificial intelligence assistant. The assistant gives helpful, detailed, and polite answers to the user's questions.
```

Comment:
it would be nice to use the recommended QA format: https://github.com/tatsu-lab/alpaca_eval/blob/b7bcc5037cc2a3585f56f94258ead693151afc1c/src/alpaca_eval/models_configs/openbuddy-llama2-70b-v10.1/configs.yaml#L5

Reply:
Yep, will rerun with instruct QA prompt format